index.js

```diff
@@ -0,0 +1,116 @@
+module.exports = {
+  env: {
+    node: true,
+    es2020: true,
+    "jest/globals": true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier/@typescript-eslint",
+    "plugin:jest/recommended",
+    "plugin:json/recommended",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.json",
+    ecmaVersion: 2020,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  plugins: [
+    "@typescript-eslint",
+    "@typescript-eslint/tslint",
+    "jest",
+    "prettier",
+    "import",
+    "json",
+  ],
+  overrides: [
+    {
+      files: ["*.js"],
+      rules: {
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/explicit-function-return-type": "off",
+      },
+    },
+    {
+      files: ["*.config.js", ".eslintrc.js", "*.json"],
+      parser: "esprima",
+    },
+  ],
+  rules: {
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        varsIgnorePattern: "^_",
```

Comment:
lets double check for vars

Reply:
Destructuring arrays is the main culprit here
```
const array = [1, 2, 3, 4]
```

We can write either:
```
const [ _one, two, _three, four ] = array
```

Or:
```
const [, two, , four] = array
```

I find the first much more readable and much less error-prone (imagine how easy it is to add an extra comma or remove one because you'd think there's too many.

Alternatively, we can disable the rule case by case basis, but this really opens up a possibility for a code that's quite hard to understand.